wiki/CLI-language.md

```diff
@@ -228,6 +278,18 @@ The automatic layers are those that are added automatically depending on the ent
 
 In addition, custom layers can be created. For this, see [Create a Layer](#create-a-layer).
 
+Layers can be used with the `ls` and `get` commands to list and to get the information of all the objects that meet the conditions of the layer. By default, only direct children of the object to which the layer belongs will be part of the response of these commands. To get results in the object hierarchy use the -r flag:
+
+```
+ls -r [layer_name]
```

Comment:
I'm not really seeing a difference between the results of `ls -r #layer` and `ls #layer`, same for get. Am I missing some use cases?